server/docker/docker-compose.yml

```diff
@@ -1,11 +1,9 @@
-version: '3.8'
-
 services:
   block-node-server:
     container_name: block-node-server
     image: block-node-server:${VERSION}
     env_file:
-      - .env
+      - ../build/.env
```

Comment:
If you need `..` to get to `build`, then either the working folder is wrong (escapes build), or we have a bug somewhere.
Docker build, in particular, should never be run from outside the `build` folder, as everything intended to be distributed _must_ be in build.
(even files that are not compiler-generated, the gradle build process should be copying them into the build folder; not a docker process).

Reply:
Thanks for that insight, initially, I did not see it in that light, but copying the `docker` dir into the project `build` dir and then containing all shell executions from `build` as a working directory is a lot safer and cleaner. I have made some changes.